libbeat/cfgfile/cfgfile.go

```diff
@@ -15,6 +15,7 @@ var (
 	// when this variable is created. See ChangeDefaultCfgfileFlag which should
 	// be called prior to flags.Parse().
 	configfiles = flagArgList("c", "beat.yml", "Configuration file")
+	overwrites  = common.NewFlagConfig(nil, nil, "E", "Configuration overwrite")
```

Comment:
Yes. Functionality provided by go-ucfg. Yes I need to document ucfg capabilities ;)